kuma/payments/urls.py

```diff
@@ -3,13 +3,8 @@
 from . import views
 
 lang_urlpatterns = [
-    re_path(r"^recurring/$", views.contribute, name="recurring_payment_initial"),
-    re_path(
-        r"^recurring/subscription/?$",
-        views.contribute,
-        name="recurring_payment_subscription",
-    ),
     re_path(r"^terms/?$", views.payment_terms, name="payment_terms"),
+    re_path(r"^thank-you/?$", views.thank_you, name="thank_you"),
```

Comment:
I think we should be consistent about the trailing slashes. I'm perfectly fine with ALWAYS having a trailing `/` by default. The only reason to use `/?` is if you have some historical legacy.

Reply:
Good point-- I changed the urls to always have a trailing `/`